invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -385,11 +385,14 @@ def deposit_create(community=None):
         community_theme = community.get("theme", {})
 
     community_use_jinja_header = bool(community_theme)
-
+    dashboard_uploads_route = current_app.config["APP_RDM_USER_DASHBOARD_ROUTES"][
```

Comment:
Hey, thinking out loud, can we maybe pass down the `dashboard` routes object and  just use the `uploads` subkey on the places that needed it? I am thinking e.g. you might need in some point to connect to `/me/communities` so it would be more flexible to have the information available. What do you think?

Reply:
```suggestion
    dashboard_routes = current_app.config["APP_RDM_USER_DASHBOARD_ROUTES"]
```

Reply:
yeah sounds good! I tested & changed it